spec/classes/candlepin_spec.rb

```diff
@@ -58,9 +58,12 @@
         end
 
         it { is_expected.to contain_file('/usr/share/tomcat/conf/login.config') }
-        it { is_expected.to contain_file('/usr/share/tomcat/conf/cert-users.properties') }
         it { is_expected.to contain_file('/usr/share/tomcat/conf/cert-roles.properties') }
         it { is_expected.to contain_file('/usr/share/tomcat/conf/conf.d/jaas.conf') }
+        it do
+          is_expected.to contain_file('/usr/share/tomcat/conf/cert-users.properties').
+            with_content('katelloUser=CN=ActiveMQ Artemis Client, OU=Artemis, O=ActiveMQ, L=AMQ, ST=AMQ, C=AMQ')
```

Comment:
I think this is needed to make the tests happy
```suggestion
            with_content("katelloUser=CN=ActiveMQ Artemis Client, OU=Artemis, O=ActiveMQ, L=AMQ, ST=AMQ, C=AMQ\n")
```